hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -94,6 +99,47 @@ def settings_schema(self):
         }
         return schema
 
+    def _rename_abc_top_group(self, abc_file, rename_to):
+        """
+        This function renames the "top" group of the alembic heirarchy.
+        Please dont confuse this top with the alembic root, which will always be named 'ABC'
+        The "top" here, is the first group in heirarchy of exported scene.
+        NOTE: This function does not recursively go through the entire scene and only renames the first group encountered in the chain.
+
+        :param string abc_file: Path of the alembic file to be modified.
+        :param string rename_to: The new name that you would want to assign to the top group.
+        """
+        import tempfile
+
+        if not os.path.exists(abc_file):
+            self.logger.error("Invalid Alembic path provided. Aborting the process "
+                              "-- {0}".format(abc_file)
+                              )
+            return
+
+        tmp_file = tempfile.NamedTemporaryFile(mode='w+b',
+                                               suffix='',
+                                               prefix='tmp',
+                                               dir=os.path.dirname(abc_file),
+                                               delete=True)
+
+        tmp_file_name = "{0}.abc".format(tmp_file.name)
```

Comment:
You can use suffix='.abc' and avoid this line.
If you just need a temp path in the same folder and don't use the file handle, there is no need to even use a NamedTemporaryFile. delete=True doesn't make sense here either. 
I suggested a temp file under the impression that we would export a temp file, then export the actual file from cask...the other way around doesn't make sense.

Reply:
ok, yeah makes sense. Made the modification to address these changes.